inst/rmarkdown/templates/wellcomeor/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,204 @@
+---
+title: Wellcome Open Research Article Template
+titlenote: The title should be detailed enough for someone to know whether the article would be of interest to them, but also concise. Please ensure the broadness and claims within the title are appropriate to the content of the article itself.
+author:
+  - name: Author Name1
+    affil: 
+    - 1
+    - 2
+    email: author1@email.com
+  - name: Author Name2
+    affil: 2
+    email: author2@email.com
+    corresponding: TRUE
+address:
+  - code: 1
+    address: Address1
+  - code: 2
+    address: Address2
+include-before: |
+  Please list all authors that played a significant role in the research involved in the article. Please provide full affiliation information (including full institutional address, ZIP code and e-mail address) for all authors, and identify who is/are the corresponding author(s).
+abstract: |
+  Abstracts should be up to 300 words and provide a succinct summary of the article. Although the abstract should explain why the article might be interesting, care should be taken not to inappropriately over-emphasise the importance of the work described in the article. Citations should not be used in the abstract, and the use of abbreviations should be minimized. If you are writing a Research or Systematic Review article, please structure your abstract into Background, Methods, Results, and Conclusions.
+keywords: |
+  Please list up to eight keywords to help readers interested in your article find it more easily.
+
+date: "`r Sys.Date()`"
+bibliography: sample.bib
+biblio-style: "unsrtnat"
+link-citations: TRUE
+colorlinks: FALSE # See hypersetup in template.tex for link color options
+# You can download the csl file if preferred and use it locally with `csl: elsevier-vancouver.csl`
+csl: https://www.zotero.org/styles/elsevier-vancouver
```

Comment:
CSL is useful only when Pandoc citeprocessing is used. Your default is `natbib` so let's include but comment

```suggestion
# Setting CSL will be necessary if you use `citation_packahe = "default"`
# You can download the csl file if preferred and use it locally with `csl: elsevier-vancouver.csl`
# csl: https://www.zotero.org/styles/elsevier-vancouver
```